lib/features/buildings_view/building_tile.dart

```diff
@@ -27,7 +29,9 @@ class BuildingTile extends ConsumerWidget {
       subtitle: context.changeNull(building.addressFormatted),
       isActive: isActive,
       onTap: () {
-        ref.read(buildingsMapControllerProvider.notifier).onMarkerTap(building);
+        unawaited(
```

Comment:
Is this unawaited here because of linter complaints, or sth changed in whole logic that now it's `async`?

Reply:
it wasn't async and now it is due to few reasons:
1. we do not have controller straight away, we use `Completer` and await its completion
2. animation also takes time and is async, and previously camera jump wasn't async